test/e2e/webview.test.ts

```diff
@@ -0,0 +1,30 @@
+import { promises as fs } from "fs"
+import * as path from "path"
+import { describe, test, expect } from "./baseFixture"
+
+describe("Webviews", ["--disable-workspace-trust"], {}, () => {
+  test("should preview a Markdown file", async ({ codeServerPage }) => {
+    // Create Markdown file
+    const heading = "Hello world"
+    const dir = await codeServerPage.workspaceDir
+    const file = path.join(dir, "text.md")
+    await fs.writeFile(file, `# ${heading}`)
+    await codeServerPage.openFile(file)
+
+    // Open Preview
+    await codeServerPage.executeCommandViaMenus("Markdown: Open Preview to the Side")
+    // Wait for the iframe to open and load
+    await codeServerPage.page.waitForTimeout(2000)
+    await codeServerPage.waitForTab(`Preview ${file}`)
```

Comment:
The iframe takes a second to load which is why I have this timeout here. Not the greatest but unsure if there's a better option.

Reply:
If we use the frame locator maybe Playwright will wait for it to appear like with other selectors?

```
page.frameLocator('#my-frame').locator(...);
```

https://playwright.dev/docs/api/class-framelocator

Reply:
I didn't try it using `"iframe"` as the selector. Good idea!